src/test/java/org/cactoos/io/InputStreamOfTest.java

```diff
@@ -46,10 +48,16 @@
  */
 @SuppressWarnings("PMD.TooManyMethods")
 public final class InputStreamOfTest {
```

Comment:
@krzyk  You have missed in this class usage of `File.createTempFile` in methods: `readsFileContent`, `readsFromUri`, `readsFromUrl`, `readsFromTextWithCharset`.

Reply:
@SharplEr Good catch, fixed.